packages/hash/frontend/src/components/BlockLoader/BlockLoader.tsx

```diff
@@ -91,7 +91,7 @@ export const BlockLoader: FunctionComponent<BlockLoaderProps> = ({
   const { readonlyMode } = useReadonlyMode();
   const { aggregateEntityTypes } =
     useBlockProtocolAggregateEntityTypes(accountId);
-  const { aggregateEntities } = useBlockProtocolAggregateEntities(accountId);
+  const { aggregateEntities } = useBlockProtocolAggregateEntities();
```

Comment:
I assume switching out the implementation for `aggregateEntities` would make some blocks crash on the frontend rather than the backend - not that they were working before. 

I think it's good that we're removing the old methods to get accurate errors from blocks (We know that everything is broken right now though :sweat_smile: )